README.md

```diff
@@ -137,16 +135,21 @@ private var splash: SplashRenderer? = null
     }
 }
 ```
-The most important annotation after @Mixin is the @SourceInjection, which tells you *where* you're putting the code.
-The options for this are:
-
-DURGAN!!!
+The most important annotation after @Mixin is the @SourceInjection (Or field or method injection), which tells you *where* you're putting the code.
```

Comment:
Take out or field/method injection

Reply:
Specifically @SourceInjection denotes where the given instructions are going in the target method in the target class. With `point` you can define what part of the method this targets.

Reply:
done